package.json

```diff
@@ -5,7 +5,9 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon ./server.js"
+    "start": "nodemon ./server.js",
+    "sass": "sass ./public/src/css/style.scss ./public/src/css/style.css --style=compressed --no-source-map",
+    "build": "tsc --build public/tsconfig.json && sass ./public/src/css/style.scss ./public/src/css/style.css --style=compressed --no-source-map"
```

Comment:
Another small suggestion... sometimes it can be useful to examine the stylesheet in the console when debugging. Perhaps we should just remove the `--style-compressed` and `--no-source-map` for now while developing and then add them back once we are done to reduce page load time.

I will approve either way though, up to you.